Module not found: Error: Can't resolve './crypto/build/Release/sshcrypto.node'

Higher version than **v1.17.0** it's not possible to debug/compile on Windows OS

Error:

`
WARNING in ./node_modules/ssh2/lib/protocol/crypto.js 30:12-60
Module not found: Error: Can't resolve './crypto/build/Release/sshcrypto.node' in 'C:\Code\GitHub\vscode-docker\node_modules\ssh2\lib\protocol'
 @ ./node_modules/ssh2/lib/server.js 27:29-60
 @ ./node_modules/ssh2/lib/index.js 33:10-32
 @ ./node_modules/docker-modem/lib/ssh.js 1:13-35
 @ ./node_modules/docker-modem/lib/modem.js 6:8-24
 @ ./node_modules/dockerode/lib/docker.js 2:10-33
 @ ./src/docker/DockerodeApiClient/DockerodeUtils.ts 17:18-38
 @ ./src/docker/DockerodeApiClient/DockerodeApiClient.ts 28:25-52
 @ ./src/docker/ContextManager.ts 119:73-123
 @ ./src/extension.ts 27:25-59
`

1 warning has detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.65.0 compiled with 1 warning in 28095 ms

Task `webpack --watch` never complete successfully and **Extension Development Host** never get opened. 

Windows version: Windows 11 Enterprise, 21H2
Node version: v16.13.1
npm version: 8.1.2

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -114,6 +114,11 @@ const config = {
             module: /node_modules\/ssh2/,
             message: /Can't resolve 'cpu-features'/
         },
+        {
+            // Ignore another warning for missing optional dependency of `ssh2`, if VS build tools aren't installed
+            module: /node_modules\/ssh2/,
+            message: /Can't resolve '.\/crypto\/build\/Release\/sshcrypto.node'/
+        },
         (warning) => false, // No other warnings should be ignored
     ],
 };

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -68,7 +68,6 @@ const config = {
     },
     plugins: [
         // Copy some needed resource files from external sources
-        // @ts-expect-error: Spurious type incompatibility error
         new CopyPlugin({
             patterns: [
                 './node_modules/vscode-azureextensionui/resources/**/*.svg',
@@ -124,7 +123,6 @@ const config = {
 };
 
 if (debugWebpack) {
-    // @ts-expect-error: Spurious type incompatibility error
     config.plugins.push(new BundleAnalyzerPlugin({ analyzerMode: 'static' }));
     console.log('Config:', config);
 }

